templates/kubernetes/index.html

```diff
@@ -376,9 +372,68 @@ <h2>Fully-managed Kubernetes</h2>
   </div>
 </section>
 
-<div class="u-fixed-width">
-  <hr>
-</div>
+<section class="p-strip--suru">
+  <div class="row">
+    <h2>Kubernetes resources</h2>
+  </div>
+  <div class="row p-divider is-dark">
+    <div class="col-4 p-divider__block">
+      <div class="p-heading-icon--small">
+        <div class="p-heading-icon__header">
+          {{
+            image(
+            url="https://assets.ubuntu.com/v1/fc781c43-Case+study+-+white.svg",
+            alt="",
+            width="32",
+            height="28",
+            hi_def=True,
+            loading="lazy",
+            ) | safe
+          }}
+          <p class="p-heading--4">Case study</p>
+        </div>
+      </div>
+      <a class="p-link--inverted" href="/engage/atresmedia-charmed-kubernetes-case-study">Atresmedia dominates Spain’s OTT media market with Charmed Kubernetes</a>
+    </div>
+    <div class="col-4 p-divider__block">
+      <div class="p-heading-icon--small">
+        <div class="p-heading-icon__header">
+          {{
+            image(
+            url="https://assets.ubuntu.com/v1/271921af-Datasheet+-+white.svg",
+            alt="",
+            width="32",
+            height="28",
+            hi_def=True,
+            loading="lazy",
+            ) | safe
+          }}
+          <p class="p-heading--4">Whitepaper</p>
+        </div>
+      </div>
+      <a class="p-link--inverted" href="/engage/kubernetes-deployment-enterprise-whitepaper">Five strategies to accelerate Kubernetes deployment in the enterprise</a>
+    </div>    
+    <div class="col-4 p-divider__block">
+      <div class="p-heading-icon--small">
+        <div class="p-heading-icon__header">
+          {{
+            image(
+            url="https://assets.ubuntu.com/v1/271921af-Datasheet+-+white.svg",
+            alt="",
+            width="32",
+            height="28",
+            hi_def=True,
+            loading="lazy",
+            ) | safe
+          }}
+          <p class="p-heading--4">Whitepaper</p>
+        </div>
+      </div>
+      <p><a class="p-link--inverted" href="https://ubuntu.com/engage/bare-metal-kubernetes">How to run workloads on bare metal Kubernetes with MAAS</a></p>
+      <p><a href="/kubernetes/resources" class="p-link--inverted">Find more resources&nbsp;&rsaquo;</a></p>
```

Comment:
The page linked to this doesn't exist.

Reply:
Good spot, this page isn't live yet so I will create a separate [issue](https://github.com/canonical-web-and-design/web-squad/issues/5110) to add it once /resources is live